webapp/advantage/views.py

```diff
@@ -6,14 +6,14 @@
 
 from dateutil.parser import parse
 import flask
-import talisker.requests
 import pytz
+from flask import g
```

Comment:
You can also do `from flask.g import api` if that's the only thing you are going to use

Reply:
It would have been nice if I could do it. But `g.api` is a global value I defined in the decorator. You can set globals with any name using `flask.g`. But I cannot import `api`, as it is not a package/variable.